br/pkg/stream/table_mapping.go

```diff
@@ -515,10 +533,12 @@ func ExtractValue(e *kv.Entry, cf string) ([]byte, error) {
 		if err := rawWriteCFValue.ParseFrom(e.Value); err != nil {
 			return nil, errors.Trace(err)
 		}
-		if rawWriteCFValue.HasShortValue() {
-			return rawWriteCFValue.shortValue, nil
+		// consistent with rewrite_meta_rawkv.go
+		if rawWriteCFValue.IsDelete() || rawWriteCFValue.IsRollback() || !rawWriteCFValue.HasShortValue() {
```

Comment:
this is the fix, looks like if we don't return early of these types the values can be `p/xxxx/xxxx` and not a valid one to parse TableInfo